# Diff: https://project-skyscraper.com/wp-json/wp/v2/categories?per_page=100
# File: api/wp-json/wp/v2/categories_per_page_100.json
# Timestamp: 2026-05-31 06:30:19 UTC
# Lines: 47 old -> 47 new
# Beautified: yes
--- old/api/wp-json/wp/v2/categories_per_page_100.json
+++ new/api/wp-json/wp/v2/categories_per_page_100.json
@@ -1,7 +1,7 @@
 [
   {
     "id": 6325,
-    "count": 38,
+    "count": 39,
     "description": "",
     "link": "https://project-skyscraper.com/category/non-classe/",
     "name": "Non classé",
